Wikipedia/Code/Article as a Living Document/ArticleAsLivingDocLargeEventCollectionViewCell.swift

```diff
@@ -14,7 +14,7 @@ class ArticleAsLivingDocLargeEventCollectionViewCell: CollectionViewCell {
     private let timestampLabel = UILabel()
     private let userInfoTextView = UITextView()
     private lazy var thankButton: AlignedImageButton = {
-        let image = UIImage(named: "thank")
+        let image = UIImage(named: "thank-unfilled")?.withRenderingMode(.alwaysTemplate)
```

Comment:
Same rendering mode to asset catalog comment.

Reply:
Updated, thanks.